e2e/CoMap/tests/TestInput.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("CoMap - TestInput", () => {
+  test("should keep the cursor position when typing", async ({ page }) => {
+    await page.goto("/test-input");
+
+    await page.getByRole("textbox").fill("Hello Jazz!");
+
+    await page.getByRole('textbox').press('ArrowRight');
+    await page.getByRole('textbox').pressSequentially(' my love');
+
+    await expect(page.getByRole('textbox')).toHaveValue("Hello Jazz my love!");
```

Comment:
The test currently outputs `Hello Jazz !my love` because the cursor is resetted after the first keydown.